tests/test_changes.bats

```diff
@@ -7,7 +7,8 @@ SECOND_FILE_TO_HIDE="$TEST_SECOND_FILENAME"
 THIRD_FILE_TO_HIDE="$TEST_THIRD_FILENAME"
 FILE_NON_EXISTANT="NO-SUCH-FILE"
 FILE_CONTENTS="hidden content юникод"
-
+FILE_CONTENTS_UPDATED="hidden content юникод\na_new_line"
+FILE_CONTENTS_CONFLICTS="hidden content юникод\n<<<<<<< file-on-disk\na_new_line\n=======\n>>>>>>> file-from-secret"
```

Comment:
ditto here. Not sure we need file-on-disk/file-from-secret?

Reply:
Also I don't think grep -e is guaranteed to exist (though it seems to on all our tests systems)

Reply:
The FILE_CONTENTS_CONFLICTS is a fixture containing the expected output of changes -g when there are conflicts

Reply:
yes, of course. I was commenting more on the content of the string, and if we really want/need those strings in the output,